docs/tutorials/new-component/component.tsx

```diff
@@ -35,7 +36,7 @@ export class KolNewComponent implements NewComponentAPI {
 				<div
 					class={{
 						'new-component': true,
-						open: this.state._open,
+						open: this.state._open  || false,
```

Comment:
Warum wird hier optional `false` gesetzt?

Reply:
Vue: Type boolean | undefined is not assignable to type boolean
Type undefined is not assignable to type boolean

alternativ: 

```						open: !!this.state._open, ```

Reply:
Ist mit !! im neuen PR umgesetzt

https://github.com/public-ui/kolibri/pull/5984